extensions/binding/org.eclipse.smarthome.binding.mqtt.generic/src/main/java/org/eclipse/smarthome/binding/mqtt/generic/internal/homie300/DeviceAttributes.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.binding.mqtt.generic.internal.homie300;
+
+import org.eclipse.smarthome.binding.mqtt.generic.internal.mapping.MapToField;
+import org.eclipse.smarthome.binding.mqtt.generic.internal.mapping.TopicPrefix;
+
+/**
+ * Homie 3.x Device attributes
+ *
+ * @author David Graeff - Initial contribution
+ */
+@TopicPrefix
+public class DeviceAttributes {
+    public enum ReadyState {
+        unknown,
```

Comment:
Shouldn't these enum values be in upper case?

Reply:
yes